build.gradle

```diff
@@ -137,6 +137,11 @@ allprojects {
   repositories {
     mavenCentral()
     mavenLocal()
+    maven {
+      url "https://repository.apache.org/content/groups/staging/"
+      mavenContent {
+      }
+    }
```

Comment:
will remove when parquet 1.13.1 is officially released

Reply:
1.13.1 is out: https://parquet.apache.org/blog/2023/05/18/1.13.1/

🥳